src/app/public/modules/input-box/input-box.module.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  CommonModule
+} from '@angular/common';
+
+import {
+  SkyInputBoxComponent
+} from './input-box.component';
+
+@NgModule({
+  declarations: [
+    SkyInputBoxComponent
+  ],
+  imports: [
+    CommonModule
+  ],
+  exports: [
+    SkyInputBoxComponent
+  ]
+})
+export class SkyInputBoxModule { }
```

Comment:
Just confirming that you do not wish to provide the`SkyInputBoxHostService` in this module?

Reply:
Correct; that service is provided by the `SkyInputBoxComponent` so that child components can place their templates into the parent input box programmatically.